src/components/controller/loader.ts

```diff
@@ -0,0 +1,49 @@
+import { IDataSources } from "../view/sources/sources";
+
+export type Callback<T> = (data: T) => void;
+class Loader {
+    baseLink: string;
+    readonly options: { [key: string]: string };
+
+    constructor(baseLink: string, options: { [key: string]: string }) {
+        this.baseLink = baseLink;
+        this.options = options;
+    }
+
+    public getResp(
```

Comment:
It's up to you to add explicitly or not a public modifier, but please be consistent.

Reply:
Dealt with access modifiers, fixed.